src/net/i2p/crypto/eddsa/spec/EdDSANamedCurveTable.java

```diff
@@ -14,7 +14,7 @@
  *
  */
 public class EdDSANamedCurveTable {
-    public static final String CURVE_ED25519_SHA512 = "ed25519-sha-512";
+    public static final String CURVE_ED25519_SHA512 = "SHA512withEd25519";
```

Comment:
- Needs changing in the `package.html` too.
- This will probably cause breakage of downstream dependents on upgrade, but simple enough for them to resolve the issue.